1.segment-data/README.md

```diff
@@ -0,0 +1,90 @@
+# Segment Cell Painting Images
+
+In this module, we present our pipeline for segmenting nuclei and cyoplasm from the Cell Health data.
+
+### Segmentation
+
+We use the CellPose segmentation algorithm to segment the nuclei from the Hoechst channel of each Cell Painting image.
+CellPose was first introduced in [Stringer, C., Wang, T., Michaelos, M. et al., 2020](https://doi.org/10.1038/s41592-020-01018-x) and we use the [python implementation](https://github.com/mouseland/cellpose).
+
+Stringer et al. trained the CellPose segmentation models on a diverse set of cell images and is therefore a good selection for our use case.
+The CellPose python implementation was particularly useful for building reproducible pipelines.
+
+### Nuclei segmentation
+
+After manually experimenting with CellPose on about 10 Cell Health nuclei images (various cell lines), we settled on the following parameters for CellPose nuclei segmentation:
```

Comment:
optional, but it might be nice to elaborate a bit more on what you mean by "manually experimenting" - what were your eyes drawn to in determining how effective different configurations were?